bob-common/src/mapper.rs

```diff
@@ -116,24 +122,26 @@ impl Virtual {
     }
 
     pub fn get_support_nodes(&self, key: BobKey, count: usize) -> Vec<&Node> {
+        debug_assert!(count < self.nodes.len());
```

Comment:
In theory it is possible, that `count == self.nodes.len()`
```suggestion
        debug_assert!(count <= self.nodes.len());
```